.github/workflows/update-payload-processing.yml

```
@@ -0,0 +1,166 @@
+          IMAGE="quay.io/opendatahub/odh-ai-gateway-payload-processing:${LATEST}"
+
+          git checkout -b "$BRANCH"
+          git add deployment/overlays/odh/params.env \
+            deployment/base/maas-controller/default/params.env \
+            deployment/base/payload-processing/manager/kustomization.yaml \
+            maas-controller/pkg/platform/tenantreconcile/constants.go \
+            test/e2e/scripts/local-deploy.sh
+          git commit -m "$(cat <<EOF
+chore: update payload-processing pin to ${SHORT}
+
+Bump ai-gateway-payload-processing from ${CURRENT:0:12} to ${SHORT}.
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)